tutorials/nats-rdb-offload/index.mdx

```diff
@@ -0,0 +1,229 @@
+content:
+  h1: Leverage on NATS Messaging Pub/Sub capabilities to offload your Managed Database.
+  paragraph: Learn how to construct a pub/sub architecture that can help you manage the load coming to your database. Dive into this Python tutorial for step-by-step guidance on asynchronous data processing.
+categories:
+  - functions
+  - messaging
+  - postgresql-and-mysql
+tags: terraform scraping-architecture
+dates:
+  validation: 
+  posted: 
+---
+
+## Introduction
+
+In this tutorial, we show how to set up a system that receives a message from a publisher and sends it to a managed database through a buffer subscriber. To do so, we  use Scaleway serverless products and deploy two applications:
+ - A publisher application, that sends a message to a NATS stream created with Scaleway Messaging and Queuing.
+ - A subscriber application, that consumes messages published to the stream and then writes the data into a Scaleway Managed Database.
+
+<Lightbox src="scaleway-scraping-architecture.webp" alt="" />
+
+We show how to provision all the required Scaleway resources using code examples and Scaleway Console [console](https://console.scaleway.com/). The code for the functions is written in Python.
+
+This project exemplifies a decoupled architecture, where producer and consumer perform specific tasks independently. This kind of design is modular and allows for flexibility and scalability. It also adheres to the principles of microservices and serverless architectures, where individual functions or scripts focus on specific tasks.
+
+To properly follow this tutorial, we recommend you to create a dedicated project in your Scaleway account.
+You can name it "RDB and NATS Tutorial". It will hold all your Scaleway ressources created for this tutorial, they will be easier to find.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Installed [NATS CLI](https://github.com/nats-io/natscli) on your local machine
+- Generated NATS Credentials (and save them under your project directory)
+- Created RDB (https://www.scaleway.com/en/docs/managed-databases/postgresql-and-mysql/quickstart/) with mySQL
+
+
+## Create a NATS server
+
+You can refer to the following documentation (https://www.scaleway.com/en/docs/serverless/messaging/api-cli/nats-cli/) to create your NATS stream.
+For the stream specifications, we recommend you to be as minimalist as possible. 
+There is no need for replication or specific parameters for this tutorial.
+
+Once you followed the abovementionned documentation, follow the next steps to set up your nats broker
```

Comment:
```suggestion
Once you have created your NATS scream by following the documentation linked above, follow the next steps to set up your NATS broker.
```

Reply:
NATS "stream"

Reply:
Rm ", follow the next steps to set up your NATS broker"